utils/python/CIME/system_test.py

```diff
@@ -361,15 +361,104 @@ def _create_newcase_phase(self, test):
     def _xml_phase(self, test):
     ###########################################################################
 
-        test_case = CIME.utils.parse_test_name(test)[0]
-        envtest = EnvTest(self._get_test_dir(test))
+        # modify the CASEROOT xml files depending on the settings of case_opts from
+        # then call case.flush()
 
+        # Determine list of component classes that this coupler/driver knows how
+        # to deal with. This list follows the same order as compset longnames follow.
         files = Files()
         drv_config_file = files.get_value("CONFIG_DRV_FILE")
-        logger.debug("Found drv_config_file %s" % drv_config_file)
-
         drv_comp = Component(drv_config_file)
+        component_classes = drv_comp.get_valid_model_components()
+
+        # Set up a case object
+        case = Case(self._get_test_dir(test)) 
+
+        # Determine the test_case from the test name
+        test_case = CIME.utils.parse_test_name(test)[0]
+        case_opts = CIME.utils.parse_test_name(test)[1] 
+
+        # Determine case_opts from the test_case
+        if case_opts is not None:
+            logger.info("case_opts are %s " %case_opts) 
```

Comment:
Please change all these logger.info to logger.debug. We want to keep the output of create_test nice and clean in the default case.